linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing/tests/correct_stig.pass.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+echo "max-test-user:x:10020:10020:Test User for Password Expiration:/:/sbin/nologin" >> /etc/passwd
+echo "max-test-user:!!:18648:1:60:::" >> /etc/shadow
```

Comment:
Since we defined "interactive users" as "users with a password", we don't actually need to hit `/etc/passwd` at all. I now flush all non-locked/blank password entries from `/etc/shadow` before adding my test users to `/etc/shadow`.

Reply:
You'll actually have to add `max-test-user` in `/etc/passwd`, the `chage`command in the remediation complains about not existing user:
```
[root@localhost ~]# chage -l max-test-user | awk -F: '/Maximum/ {print $2}'
chage: user 'max-test-user' does not exist in /etc/passwd
```